.github/workflows/python_compile_requirements.yml

```diff
@@ -31,6 +29,6 @@ jobs:
           add-paths: python/
           title: Compile python requirements
           author: github-actions[bot] <github-actions[bot]@users.noreply.github.com>
-          assignees: fvankrieken, damonmcc, alexrichey, sf-dcp
+          assignees: ${{ github.event_name == 'schedule' && 'fvankrieken, damonmcc, alexrichey, sf-dcp' || '' }}
```

Comment:
Something was weird in my dev container so ran this via action. Figured in these cases when the PR is generated we don't need to assign us